toolkits/global/packages/global-forms/view/label.hbs

```diff
@@ -0,0 +1,6 @@
+<label for="{{id}}" class="c-forms__label">
+	{{label}}{{#if optional}} (optional){{/if}}
```

Comment:
🤔  I don't think "optional" is really helpful for the following reasons

1. By default all text fields are "optional" i.e they dont have "required" set to true.
2. This will not be useful if the service has to support multiple languages because the copy is not translated.

When I worked in GOV we tried to make all fields required by default (no point asking a user for data if its not useful for all) and only flag up "optional"  fields in the label like what is done here but in our case the text fields are not required by default so seems to send mix messages as to when to use this option or not.

Reply:
Instead just let the devs control the copy and if they want to use "(Optional)" they could just include it in their copy that they need for the label.

Reply:
The determination of what is optional or not, or what copy should be used would come from agreement across dsign/ux/content that would ultimately end up in the Design System docs site?

Reply:
>Instead just let the devs control the copy and if they want to use "(Optional)" they could just include it in their copy that they need for the label.

From a content design point of view, I'd really like to introduce some standardisation if possible. I get that we're not currently offering localisation of this copy, but we're not currently doing that for the content elements of any components so this isn't a new problem. My concern is if we don't include something, people are going to fall back to using asterisks.

Reply:
So, @Heydon has followed the Sketch designs (see screenshot). I'd have to check with James (on holiday until Monday (11/07/22) but I think all of the form designs have gone through rounds of review in SN digital and this is currently the latest agreed across discipline.

![Screenshot 2022-07-05 at 11 45 12](https://user-images.githubusercontent.com/925197/177311077-8def55e6-e2fb-443b-9c84-34fdad2107a2.png)

Reply:
regarding the issue with translation. It could be made a handlebars variable in the json, rather than hardcoded as it is now‽ So it could get switched out as needed? (pinging @Heydon) 

[Microcopy](https://www.microcopy.me) like (optional) here could also be tokenised -- which could further help documentation of it all

Reply:
I was just following the UX. Can we discuss this after v1 is out, please? For the time being, just put `required: true` on all your fields if you want them all required.